GetIntoTeachingApi/Services/CrmService.cs

```diff
@@ -76,6 +76,13 @@ public IEnumerable<TeachingEvent> SearchTeachingEvents(TeachingEventSearchReques
                 .OrderBy(teachingEvent => teachingEvent.StartAt);
         }
 
+        public TeachingEvent GetTeachingEvent(Guid id)
+        {
+            return GetTeachingEvents()
+                .Where((teachingEvent) => teachingEvent.Id == id)
+                .FirstOrDefault();

```

Comment:
do you need the Where() ?
can it be done in the FirstOrDefaiult() ?

FirstOrDefault((teachingEvent) => teachingEvent.Id == id)